LambdaTimeline/View Controllers/ImagePostViewController.swift

```diff
@@ -43,13 +50,16 @@ class ImagePostViewController: ShiftableViewController {
             return
         }
         
-        let imagePicker = UIImagePickerController()
-        
-        imagePicker.delegate = self
+        DispatchQueue.main.async {
+            let imagePicker = UIImagePickerController()
+            
+            imagePicker.delegate = self
+            
+            imagePicker.sourceType = .photoLibrary
+            
+            self.present(imagePicker, animated: true, completion: nil)
+        }
```

Comment:
You can group similar lines together, and use spacing when there's large blocks to give some visual space, but you don't need to do every other line.

```swift
DispatchQueue.main.async {
    let imagePicker = UIImagePickerController()
    imagePicker.delegate = self
    imagePicker.sourceType = .photoLibrary
    self.present(imagePicker, animated: true, completion: nil)
}
```